### DISCUSSION TITLE
Should we be concerned if developers sell their revenue directly to the market?

### DISCUSSION THREAD

**Discussion by Freedom-kin** [kinfoundation/kin-rfcs#149] [Spend Based KRE]
## Should we be concerned if developers sell their revenue directly to the market?

Kin sold on the market is not 'lost'.   
For every sale there is a purchase for the exact same amount.   
Kin just changes hands.

Simply changing hands doesn't necessarilly have to lower Kin's value.    
**So what does lower that value?**   
It is the supply and demand relationship.

Huge amounts of free Kin are being handed out. 
Not only through the KRE but also through grants of billions of Kin.
Also think of marketing campaigns such as in March 2022, 
in which Kucoin gives away 750 million Kin for free in just one week.

To date, Kin worth 70 Million USD has been awarded in the form of rewards.    
**That's Kin  nobody paid for.**

What is the effect of free Kin?   
How can Kin that you bought compete on an exchange with Kin that was obtained for free?   
If you sell too low you make a loss, but the one with free Kin can not make a loss.
Everything he gets for it is a win.

And so the race to the bottom has begun.   
If the market sentiment is negative, this will amplify the effect.   
Even when it is positive the uptrent is dampened.

So it is not just the fact that developers offer their Kin on the market.   
The problem is in the sheer amount of free Kin.

Lets not forget that since staking means free Kin, this also applies to regular users, not just to developers.

It is no longer in proportion to what the market absorbs, resulting in lower prices….supply and demand.
As long as the rewards are higher than new money entering the Kin Economy, there will be more supply than demand.

### How will this proposal help to better balance supply and demand?

KRE will always put Kin into circulation for free, that's the principle of the KRE.    
However, this proposal adjusts the amount of free Kin based on the Economic Activity actually generated.

It is no longer a bag of money that is handed out anyway, regardless of the economic results.

The KRE will adjust to the total net turnover of all apps.    
When apps are added, it increases, but when apps leave, the KRE decreases.   
It adapts to the actual FLOW OF KIN through the economy.

An app that brings in 30% of its net sales has really contributed to the Kin Economy. For every 1000 Kin reward, he has generated 3,333 Kin turnover. That is real economic activity.

When larger apps join the Kin Economy because they now have a real incentive (30% bonus), many more users will buy Kin.
Little by little, the original investors will be replaced by real users.

More apps = more users.    
More users = more demand.

So on the one hand we increase the demand and on the other hand we dampen the excessive amount of free Kin.
The Kin that is being rewarded actually does what it is supposed to do... Create economic activity.

All we have to do is reward net SPENDS ( net turnover ).

**Everything you reward gets bigger.**

**Comment by Rachel1-2** [comment]
I am not a developer of course, but since the developers pay a lot of their KRE payout to users (with KRE 3.0), I can talk about the user experience. 

Everything I receive for free (staking) is easier to sell. No matter if it is free KIN, Orca, Ray etc. I never paid for it so everything is profit. However when I compare that to coins/tokens I did pay for then if my investment is under water selling hurts. If it is not under water, but I didn't sell when the price was 10x higher, then again selling hurts. I don't have this problem at all with staking rewards (free KIN) and I think many with me. 

That is why I think it is a wrong assumption that rewarding AUB relieves sell pressure, because handing out free KIN does the opposite! 

Another important thing to remember is everything you get for free has little to no value for people, that is just how psychology works. When people pay for it, it has value.
